include/onnxruntime/core/common/logging/logging.h

```diff
@@ -289,12 +301,12 @@ class Logger {
 };
 
 inline const Logger& LoggingManager::DefaultLogger() {
```

Comment:
default logger should never throw, worst case would be that nothing could be logged.

Reply:
But, I feel it works like a guard. Whenever you saw an exception was thrown from here, it means you didn't create an OrtEnv  first.

Reply:
Might need that "guard" yes, definitely not here in this way.